lisp/init-javascript.el

```diff
@@ -57,7 +57,10 @@
     (add-hook 'xref-backend-functions #'xref-js2-xref-backend nil t))
   (with-eval-after-load 'js
     (define-key js-mode-map (kbd "M-.") nil)
-    (add-hook 'js-mode-hook 'sanityinc/enable-xref-js2))
+    (define-key js-ts-mode-map (kbd "M-.") nil)
+    (add-hook 'js-mode-hook 'sanityinc/enable-xref-js2)
+    (add-hook 'js-ts-mode-hook 'sanityinc/enable-xref-js2))
```

Comment:
js-mode-hook is actually run in js-ts-mode

Reply:
No, actually both js-mode and js-ts-mode inherit from js-base-mode in Emacs >= 29

Reply:
Ah yes, sorry, I had been tinkering with the code locally and my mind was thinking of `js-base-mode`.